var/spack/repos/builtin/packages/clingo/package.py

```diff
@@ -3,9 +3,13 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import sys
+
 from spack.compiler import UnsupportedCompilerFlag
 from spack.package import *
 
+is_windows = sys.platform == "win32"
```

Comment:
This is no longer used in this package (I'm guessing it was added before spec-based constraints) and likewise for `import sys`.

Reply:
You are correct, and it has been removed.